renderer/components/ActivitiesTable.tsx

```diff
@@ -279,6 +279,20 @@ export default function ActivitiesTable({
                   } else {
                     onEditActivity(activity);
                   }
+                  global.ipcRenderer.send(
+                    "send-analytics-data",
+                    "registrations",
+                    {
+                      registration: "google-calendar-event_registration",
+                    }
+                  );
+                  global.ipcRenderer.send(
+                    "send-analytics-data",
+                    "registrations",
+                    {
+                      registration: `all_calendar-events_registration`,
+                    }
+                  );
```

Comment:
in this case you catch all pressing on "edit" and "add" icon. If you want to catch only added google event, you need to send analytics inside "if (activity.calendarId)" condition in 274 line. Also `all_calendar-events_registration` doesn't correct because now we are showing only google events in activity table, so you can delete it.